policy/policy.go

```diff
@@ -44,85 +45,198 @@ func (policy *PolicySvc) Routes() common.Routes {
 		},
 		common.Route{
 			Method:          "DELETE",
-			Pattern:         "/policies/{id}",
+			Pattern:         "/policies/{policyID}",
 			Handler:         policy.deletePolicy,
 			MakeMessage:     nil,
 			UseRequestToken: false,
 		},
+		common.Route{
+			Method:          "GET",
+			Pattern:         "/policies",
+			Handler:         policy.listPolicies,
+			MakeMessage:     nil,
+			UseRequestToken: false,
+		},
 	}
 	return routes
 }
 
-// handleHost handles request for a specific host's info
-func (policy *PolicySvc) addPolicy(input interface{}, ctx common.RestContext) (interface{}, error) {
-	policyDoc := input.(*common.Policy)
-	log.Printf("Request for a new policy to be added: %v", policyDoc)
-	client, err := common.NewRestClient(common.GetRestClientConfig(policy.config))
+// augmentEndpoint augments the endpoint provided with appropriate information
```

Comment:
I think this docstring could be more helpful. :-) Maybe:

    augmentEndpoint looks up additional information about the specified endpoint object, by
    retrieving information about this endpoint from various Romana services. It then adds
    this information to the endpoint object.

    Side effect: The passed in endpoint object will change.